src/sentry/integrations/opsgenie/integration.py

```diff
@@ -146,19 +149,19 @@ def get_organization_config(self) -> Sequence[Any]:
         return fields
 
     def update_organization_config(self, data: MutableMapping[str, Any]) -> None:
-        # get the team ID/test the API key for a newly added row
+        # add the integration ID to a newly added row
+        if not self.org_integration:
+            return
+
         teams = data["team_table"]
         unsaved_teams = [team for team in teams if team["id"] == ""]
+        # this is not instantaneous, so you could add the same team a bunch of times in a row
```

Comment:
what do you mean by this? like is there a button that could be spam clicked to cause an issue?